komodoenv/statfs.py

```diff
@@ -0,0 +1,67 @@
+import sys
+from pathlib import Path
+from ctypes import (
+    CDLL,
+    Structure,
+    c_int32,
+    c_uint32,
+    c_int64,
+    c_uint64,
+    c_char_p,
+    byref,
+    create_string_buffer,
+)
+from six import ensure_binary
+
+
+# From /usr/include/linux/magic.h
+_TMPFS_MAGIC = 0x01021994
+_NFS_SUPER_MAGIC = 0x00006969
+
+
+class _Statfs(Structure):
+    """Based on Linux' `struct statfs`. Read: man 2 statfs"""
+
+    fsblkcnt_t = c_uint64
+    fsfilcnt_t = c_uint64
+    fsid_t = c_uint64
+
+    _fields_ = (
+        ("f_type", c_int64),
+        ("f_bsize", c_uint64),
+        ("f_blocks", fsblkcnt_t),
+        ("f_bfree", fsblkcnt_t),
+        ("f_bavail", fsblkcnt_t),
+        ("f_files", fsfilcnt_t),
+        ("f_ffree", fsfilcnt_t),
+        ("f_fsid", fsid_t),
+        ("f_namelen", c_int64),
+        ("f_frsize", c_uint64),
+        ("f_spare", c_int64 * 5),
+    )
+
+
+def _test_fs_type(path, f_type):
+    if sys.platform != "linux":
+        return
```

Comment:
```suggestion
        return False
```

Reply:
My idea was that `None` is not-available (eg. you're running this on a mac where `statfs` does something else). `False` is no.

The distinction really doesn't matter though and I should probably remove the check entirely. What do you think?